service/models/datasf.py

```diff
@@ -0,0 +1,17 @@
+"""DataSF model"""
+import requests
+
+class DataSF():
+    """DataSF model class"""
+    @staticmethod
+    def filter(endpoint, filters=None, headers=None):
+        """https://dev.socrata.com/docs/filtering.html"""
```

Comment:
can you elaborate on this doc string a bit?

Reply:
yup. 6b91fd8487bc55f70d7cf2ea60e16f6a9c7cf93f